### ISSUE TITLE
ggplot changed arguments

### ISSUE THREAD

**Issue by FlorianScharf** [mike-lawrence/ez#50]
Hey,

I really like your package and appreciate your efforts to make ANOVA more accessible in R. :-)

I noticed a minor issue in ezPlot: 
When you print error bars, it says:

> Warning message:
> `show_guide` has been deprecated. Please use `show.legend` instead. 

I usually just print the code and correct it manually, but maybe you could also fix it on your side.

Cheers,
Florian

**Comment by mike-lawrence** [comment]
Good catch. I haven't done any development on ez in a while (moved to a
Bayesian workflow), but I know it's a widely used package still, so I
should do better at keeping it behaving properly. I see you also just
pushed some commits that I'll take a look at this afternoon, as well as
generally check that there's no surprise incompatibilities with ez and the
coming ggplot2 update.
